examples/MaxCut/WithFlip.ipynb

```diff
@@ -38,7 +38,7 @@
    "outputs": [
```

Comment:
the changes to this file are only due to a new execution? If yes, remove it from the commit.